Coralogix/Docs/CoralogixExporterOptions.md

```diff
@@ -69,4 +69,10 @@ var labels: [String: Any]?
 ```
 Sets labels that are added to every Span. This is an optional property.
 
+### sampleRate
+```swift
+let sampleRate: Float?
```

Comment:
why is it float?
its weird to pass sample rate of 20.85

Reply:
fixed changed to Int